ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/security/trustedlaunch/cucushift-installer-rehearse-azure-ipi-security-trustedlaunch-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/security/trustedlaunch/cucushift-installer-rehearse-azure-ipi-security-trustedlaunch-workflow.yaml",
```

Comment:
How about renaming `security` to `confidential`, so that we can keep consistent with the gcp ones.

Reply:
Two security type are supported, "TrustedLaunch" and "ConfidentialVM".
If renaming `security` to `confidential` , the path for both will be `xxx-confidential-trustedlaunch` and `xxx-confidential-confidentialvm` ?

Reply:
> If renaming security to confidential , the path for both will be xxx-confidential-trustedlaunch and xxx-confidential-confidentialvm ?

If keep `security`, we still would have two install workflow, right? If right, sounds renaming the folder make no much difference.

Another reason is becasuse I saw ipi-conf-azure-confidential support both setting, so I proposed to keep naming consistent with it.

Some more ideas, if we are going to add one more install workflow for `confidentialvm`, cucushift-installer-check-azure-confidential would become complex to support both type checking, right? How about introduce two dedicated sets of `xxx-confidential-trustedlaunch` and `xxx-confidential-confidentialvm` on ipi-conf-azure and cucushift-installer-check-azure-confidential for static configurations, so that we can reduce some if conditions to be a bit simper?

Reply:
make sense, this way can make post check more easier and reduce CI failure caused by installer post check script.
Updated.